internal/auth0/organization/resource_connections_test.go

```diff
@@ -13,20 +13,56 @@ import (
 )
 
 const testAccGivenTwoConnectionsAndAnOrganization = `
-resource "auth0_connection" "my_connection_1" {
-	name     = "Acceptance-Test-Connection-1-{{.testName}}"
+resource "auth0_connection" "my_db_connection" {
+	name     = "Acceptance-Test-DB-Connection-{{.testName}}"
 	strategy = "auth0"
 }
 
-resource "auth0_connection" "my_connection_2" {
-	depends_on = [ auth0_connection.my_connection_1 ]
+resource "auth0_connection" "my_enterprise_connection" {
```

Comment:
Could we slim this supporting resource down to only essential parameters?

Reply:
I'm not entirely sure which ones are essential, I found that it required quite a lot of them to be considered valid. I will see if I can trim it down a bit

Reply:
I got it cleaned up now